packages/opentelemetry-tracing/src/export/ReadableSpan.ts

```diff
@@ -29,7 +29,7 @@ import { TimedEvent } from '../TimedEvent';
 export interface ReadableSpan {
   readonly name: string;
   readonly kind: SpanKind;
-  readonly spanContext: SpanContext;
+  readonly spanContext: () => SpanContext;
```

Comment:
```suggestion
  spanContext(): SpanContext;
```
I find this more readable and it is in sync with the API Span interface